users/views/user_invitation_view.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import render
+from django.contrib import messages
+from django.contrib.auth.models import User
+from users.forms import UserInvitationForm
+from django.views.generic import View
+from users.models import SignupInvitation
+from django.core.mail import send_mail
+
+
+class UserInvitationView(View):
+    template_name = "registration/send_invitation.html"
+
+    def get(self, request):
+        form = UserInvitationForm()
+        return render(request, self.template_name, {'form': form})
+
+    def post(self, request):
+        form = UserInvitationForm(request.POST)
+        if form.is_valid():
+            invited = SignupInvitation()
```

Comment:
Since we are using a Model Form, you can change lines 20 to 23 to simply:
invited = form.save()